libcxx/include/__random/seed_seq.h

```diff
@@ -79,6 +81,11 @@ void seed_seq::__init(_InputIterator __first, _InputIterator __last) {
 
 template <class _RandomAccessIterator>
 void seed_seq::generate(_RandomAccessIterator __first, _RandomAccessIterator __last) {
+  using _ValueType = typename iterator_traits<_RandomAccessIterator>::value_type;
+  static_assert(is_unsigned<_ValueType>::value && sizeof(_ValueType) * CHAR_BIT >= 32,
```

Comment:
Since we use `uint32_t` unconditionally in our code, this seems more readable.
```suggestion
  static_assert(is_unsigned<_ValueType>::value && sizeof(_ValueType) >= sizeof(uint32_t),
```

Reply:
100% agreed